sugoi-api-core/src/main/java/fr/insee/sugoi/core/service/impl/GroupServiceImpl.java

```diff
@@ -90,27 +94,33 @@ public void update(String realm, String appName, Group group) {
 
   @Override
   public void addUserToGroup(String realm, String userId, String appName, String groupName) {
+    User user = userService.findById(realm, null, userId);
```

Comment:
We should use this to return early if no user is found (with an exception, managed in the controller service)

Reply:
Yes but findById already throw a new EntityNotFoundException("User not found in realm " + realmName) if the user is not found. Do you think i need to try catch findById and throw a new RuntimeException ?

Reply:
no, I thought we returned null.